src/disc/stk/Albany_GenericSTKMeshStruct.cpp

```diff
@@ -1545,7 +1545,10 @@ GenericSTKMeshStruct::getValidGenericSTKParameters(std::string listname) const
   validPL->set<int>("Exodus Write Interval", 3, "Step interval to write solution data to Exodus file");
   validPL->set<std::string>("Method", "",
     "The discretization method, parsed in the Discretization Factory");
-  validPL->set<int>("Cubature Degree", 3, "Integration order sent to Intrepid2");
+
+  //TODO: remove the following after updating MPAS
+  //validPL->set<int>("Cubature Degree", 3, "Integration order sent to Intrepid2");
```

Comment:
So it can be removed already?

Reply:
oh, I need to uncomment that for now.

Reply:
Actually I removed this comment. I just checked and in all MPAS tests we rely on the default cubature degrees hard coded in the Interface